### DISCUSSION TITLE
Cannot open any of mine repositories anymore. It just returns generic error.

### DISCUSSION THREAD

**Discussion by abritov** [community/community#118422] [Mobile]
The feature worked on previous version which I had on iOS ~14
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.157.0 (127735376)
Device: iPhone SE (2nd generation) (iOS 17.4.1)
</details>

**Comment by jfullstackdev** [comment]
I can definitely open your Public Repos, so what's the problem ? 

![image](https://github.com/community/community/assets/47092464/f88d1bbe-5188-494a-961b-36412fb80a06)

**Reply by abritov** [reply]
Something wrong with mobile app, it doesn't open any repo:
![image](https://github.com/community/community/assets/12711566/9ca22496-6e2f-4a29-b62f-e6bd473bfcfb)

**Comment by eliperkins** [comment]
Hey @abritov, thanks for reaching out! Not having your repos load in the app seems quite frustrating. If you've got a specific URL to a repo that we can use to reproduce this issue, I'd love to work with our engineers here to get this addressed!

**Reply by abritov** [reply]
I'll try to get insights from app traffic, seems like something broke during update.

**Comment by abritov** [comment]
After sign in out and getting back into account bug disappeared. Reinstalling app did not work.
Problem solved
